Lights set_light_level asserts incorrect value range

```python
>>> import dirigera
>>> hub = dirigera.Hub(token="...", ip_address="xxx.xxx.xxx.xxx")
>>> light = hub.get_light_by_name("...")
```

Negative values raise an AssertionError as expected:

```python
>>> light.set_light_level(-1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/user/.local/lib/python3.11/site-packages/dirigera/devices/light.py", line 73, in set_light_level
    raise AssertionError("light_level must be a value between 0 and 100")
AssertionError: light_level must be a value between 0 and 100
```

https://github.com/Leggin/dirigera/blob/192a8483051e821ba3514e53e1b1715c6bf1239b/src/dirigera/devices/light.py#L72-L73

Zero should be ok according to the error message but raises an HTTPError:

```python
>>> light.set_light_level(0)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/user/.local/lib/python3.11/site-packages/dirigera/devices/light.py", line 76, in set_light_level
    self.dirigera_client.patch(route=f"/devices/{self.device_id}", data=data)
  File "/home/user/.local/lib/python3.11/site-packages/dirigera/hub/hub.py", line 78, in patch
    response.raise_for_status()
  File "/home/user/.local/lib/python3.11/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://xxx.xxx.xxx.xxx:8443/v1/devices/...
```

One and above work as expected:

```python
>>> light.set_light_level(1)
>>> 
```

--- a/src/dirigera/devices/light.py
+++ b/src/dirigera/devices/light.py
@@ -69,8 +69,8 @@ class Light(Device):
             raise AssertionError(
                 "This lamp does not support the set lightLevel function"
             )
-        if light_level < 0 or light_level > 100:
-            raise AssertionError("light_level must be a value between 0 and 100")
+        if light_level < 1 or light_level > 100:
+            raise AssertionError("light_level must be a value between 1 and 100")
 
         data = [{"attributes": {"lightLevel": light_level}}]
         self.dirigera_client.patch(route=f"/devices/{self.device_id}", data=data)

